feature-detects/css/pseudoanimations.js

```diff
@@ -5,11 +5,11 @@
   "tags": ["css"]
 }
 !*/
-define(['Modernizr', 'testStyles', 'prefixes', 'test/css/animations'], function(Modernizr, testStyles, prefixes) {
+define(['Modernizr', 'testStyles', 'prefixes', 'test/css/animations', 'computedStyle'], function(Modernizr, testStyles, prefixes, computedStyle) {
   Modernizr.addTest('csspseudoanimations', function() {
     var result = false;
 
-    if (!Modernizr.cssanimations || !window.getComputedStyle) {
+    if (!Modernizr.cssanimations || !computedStyle) {
```

Comment:
same here - this is `|| false`

Reply:
And this check can be removed anyway, since the check is done in the computedStyle already and would return null there...